pymc3/step_methods/gibbs.py

```diff
@@ -1,16 +1,17 @@
-'''
+"""
```

Comment:
Maybe this can be removed as well now following https://github.com/pymc-devs/pymc3/pull/2405#issuecomment-314105123